datameta/errors.py

```diff
@@ -19,25 +19,33 @@
 # LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
-from typing import List
+import enum
+from typing import List, Optional
 from pyramid.httpexceptions import HTTPBadRequest
 
 
-def get_validation_error(messages:List[str]) -> HTTPBadRequest:
+def get_validation_error(
+    messages:List[str], 
+    fields:Optional[List[Optional[str]]]
```

Comment:
Does the string need to be `Optional` in the type hint?

Reply:
Yes, it can either accept None or a List of elements that are either string or None

Reply:
I see 👍